### ISSUE TITLE
Add a button for dumping common crawl and regular dumps from Nutch

### ISSUE THREAD

**Issue by chrismattmann** [memex-explorer/memex-explorer#543]
So, if we run the command like:

```
vagrant@vagrant-ubuntu-trusty-64:~/miniconda/envs/memex/lib/nutch$ ./bin/nutch commoncrawldump -outputDir /vagrant/source/resources/crawls/commoncrawl-atf -segment /vagrant/source/resources/crawls/atf-crawl-1/segments/
vagrant@vagrant-ubuntu-trusty-64:~/miniconda/envs/memex/lib/nutch$ 
```

We can easily dump out the data from a crawl in Common Crawl Format and in regular file format. And it shows in log.io when this is happening, e.g.,:

```
nutch application_server 2015-05-27 14:22:08,954 WARN tools.CommonCrawlDataDumper - Output directory: [/vagrant/source/resources/crawls/commoncrawl-atf]: does not exist, creating it.
nutch application_server 2015-05-27 14:22:10,591 INFO tools.CommonCrawlDataDumper - Processing segment: [/vagrant/source/resources/crawls/atf-crawl-1/segments/20150527140155]
nutch application_server 2015-05-27 14:22:10,878 WARN util.NativeCodeLoader - Unable to load native-hadoop library for your platform... using builtin-java classes where applicable
nutch application_server 2015-05-27 14:22:12,798 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/9a/d3/9381cf2ba9558bd3d594ace7365f779a_.html]
nutch application_server 2015-05-27 14:22:12,886 INFO tools.CommonCrawlDataDumper - Processing segment: [/vagrant/source/resources/crawls/atf-crawl-1/segments/20150527140316]
nutch application_server 2015-05-27 14:22:13,098 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/5d/89/579161b7d9ee0c998e8f33db9ce5ef89_.html]
nutch application_server 2015-05-27 14:22:13,166 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/d9/f2/d4518a899690138df7c5e841278b1375_index.html]
nutch application_server 2015-05-27 14:22:13,240 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/1b/75/1c507609b4f473167abf362d54a09370_post.html]
nutch application_server 2015-05-27 14:22:13,324 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/fa/6e/ff09b0f6a0a2cbc76d896b74e239dc01_.html]
nutch application_server 2015-05-27 14:22:13,424 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/1f/dd/18707742f3fd90b1de31a414d94a56b9_al-com.js]
nutch application_server 2015-05-27 14:22:13,530 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/35/e1/3e740ee75e1d5da3e68b8dca170c78b1_movabletype.min.js]
nutch application_server 2015-05-27 14:22:13,598 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/05/cb/03951f6150791b05c9662715bdfc9323_contactlink.js]
nutch application_server 2015-05-27 14:22:13,669 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/b9/d0/b60d9905961b1440db4c779d04585172_classifieds_concentrated.css]
nutch application_server 2015-05-27 14:22:13,805 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/d0/ee/d2708f9408a5007de896b8afe8e7cfb5_classifieds.js]
nutch application_server 2015-05-27 14:22:13,927 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/a1/f0/a1da003b1ee9b25af64f2977042fbf1f_jquery.tooltips.js]
nutch application_server 2015-05-27 14:22:14,070 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/10/ed/1e426a2b099e7f68ee9b6636d5e47891_cookie.js]
nutch application_server 2015-05-27 14:22:14,096 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/a0/fa/a5b247ef006a9b6cfbb18e68a2cf8e29_ads.js]
nutch application_server 2015-05-27 14:22:14,166 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/b3/a6/ba7f2a253a2bdcfca1f73b936cc24423_adv-js-loader.js]
nutch application_server 2015-05-27 14:22:14,201 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/90/e4/9a662baa01d84198e1a4f85544b43413_adv_burt_bama.js]
nutch application_server 2015-05-27 14:22:14,271 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/30/2f/3c95313309a424ae2fb3c3c6f228c8b0_jquery-1.8.2.min.js]
nutch application_server 2015-05-27 14:22:14,346 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/01/5a/05c3f54c1d8433295bf35c5ca8d50a17_modernizr-2.6.2.min.js]
nutch application_server 2015-05-27 14:22:14,416 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/72/aa/720be24c2fdb9e4aa5f91b15a49b974a_plugins.js]
nutch application_server 2015-05-27 14:22:14,484 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/fc/03/f1dd53f3c37a576b06fc207b3aae0786_.html]
nutch application_server 2015-05-27 14:22:14,539 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/73/c7/77e1f47c3ce14cd1cb97e41b7c23c1cb_.html]
nutch application_server 2015-05-27 14:22:14,649 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/4e/5b/4a18c0a9ece143f35151856fb5820beb_.html]
nutch application_server 2015-05-27 14:22:14,844 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/0b/b8/037e75b1bff844d8bbd5a3dc882ab413_alphotos.html]
nutch application_server 2015-05-27 14:22:14,920 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/52/23/5f867a4c2788f4a929cf8e5e39686037_.html]
nutch application_server 2015-05-27 14:22:14,946 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/20/6d/290323880dd60ff768ccb28edd071006_for-rent.html]
nutch application_server 2015-05-27 14:22:14,975 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/ab/3b/aad9002fbd40137f3161649ebacf76aa_utag.sync.js]
nutch application_server 2015-05-27 14:22:15,024 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/70/20/76b951910c5c6c122cbadb76010fa514_.html]
nutch application_server 2015-05-27 14:22:15,081 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/2a/97/2becc591a132cb8d90122fa5794ef89b_regions_bar.js]
nutch application_server 2015-05-27 14:22:15,252 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/5c/ba/5e655208c9a78d6cbcbf1c70ab4a0ecd_alabama.html]
nutch application_server 2015-05-27 14:22:15,357 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/61/53/66ed0300177a66d654bcc8cb39001e92_.html]
nutch application_server 2015-05-27 14:22:15,450 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/69/b1/6525387f9b989a9bb91481c11861cb76_auburn.html]
nutch application_server 2015-05-27 14:22:15,517 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/c0/c3/c9c1be4d05117031c9e561be392dd139_auburnfootball.html]
nutch application_server 2015-05-27 14:22:15,606 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/db/c6/defd2fa8beb5c371cf638d13630699c5_.html]
nutch application_server 2015-05-27 14:22:15,712 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/04/ba/03ce9a99471035eebaace14aa008eece_bhammag.html]
nutch application_server 2015-05-27 14:22:15,772 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/92/e8/9bdb66982558cfafe743b2dc8627a192_.html]
nutch application_server 2015-05-27 14:22:15,836 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/88/ee/8ee3f4008f915d1fe4e7b523e98aad4c_.html]
nutch application_server 2015-05-27 14:22:15,941 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/08/c6/056ad0f88f40bd5ccf4763f463627128_careeradvice.html]
nutch application_server 2015-05-27 14:22:16,146 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/b6/83/b12acfd666aee2048e1c1c513e33b235_.html]
nutch application_server 2015-05-27 14:22:16,199 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/20/2b/2ff477b609af2a2c2b5c5242b5904913_.html]
nutch application_server 2015-05-27 14:22:16,295 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/d2/60/d9af82032462714f69bd234505fcca39_.html]
nutch application_server 2015-05-27 14:22:16,423 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/fa/3b/ff71fc7ca64262d831192796b31933c3_college-football.html]
nutch application_server 2015-05-27 14:22:16,516 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/d7/e1/dab60fe57039caf6e2620aea10e0f851_.html]
nutch application_server 2015-05-27 14:22:16,629 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/5b/ed/53a1e00ab1105daced5239fcde6df9f6_regions.css]
nutch application_server 2015-05-27 14:22:16,692 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/1e/80/1cd9d94ce8e26d1d8ee6250c00b729f3_data.html]
nutch application_server 2015-05-27 14:22:16,748 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/05/09/01dcd7cd57e9127d051b4ba29314f1e0_.html]
nutch application_server 2015-05-27 14:22:16,786 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/49/3a/4f4940049a2cfcd93d78466caa77e019_.html]
nutch application_server 2015-05-27 14:22:16,831 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/9d/46/93be5bb4d58c11a34df4790766e2d9e8_.html]
nutch application_server 2015-05-27 14:22:16,880 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/f4/54/f1a278e74996e58a56f3930941d00535_events.html]
nutch application_server 2015-05-27 14:22:16,951 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/18/9f/1fa391a78f3eb9ad9b6a29a4f3f23aee_.html]
nutch application_server 2015-05-27 14:22:17,007 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/80/93/89040f780656fbd39fbd917536c19900_favicon.ico]
nutch application_server 2015-05-27 14:22:17,045 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/9f/73/9e1efeeafa9866f871565eb030032b39_.html]
nutch application_server 2015-05-27 14:22:17,060 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/70/50/739b97460279b0365a52eb9c01d0e51f_.html]
nutch application_server 2015-05-27 14:22:17,080 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/27/73/224ebb5f740d00fa7524c413380ac0e6_.html]
nutch application_server 2015-05-27 14:22:17,159 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/11/cd/1c3f7b0010d2173bc3d6f5efdf831a86_.html]
nutch application_server 2015-05-27 14:22:17,193 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/83/18/876e967d3a20bb6017fcd38289562806_.html]
nutch application_server 2015-05-27 14:22:17,243 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/90/4b/94ac77020e993239427a9c71b21e50a1_home-garden.html]
nutch application_server 2015-05-27 14:22:17,310 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/84/a7/8c8373ab4f91ffd6a4e63efa73b23c2a_.html]
nutch application_server 2015-05-27 14:22:17,372 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/fc/d2/f049f5ccc9e3fb0adc4c459b2dc2bbc3_investigates.html]
nutch application_server 2015-05-27 14:22:17,433 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/c1/8e/cb0e47241b7a42908506d646e5c94318_jobnews.html]
nutch application_server 2015-05-27 14:22:17,511 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/33/94/36d2e617360bfae29270162a46b9f9d9_.html]
nutch application_server 2015-05-27 14:22:17,583 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/46/82/4e59ae806d75a987842b584a2fb03e36_.html]
nutch application_server 2015-05-27 14:22:17,645 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/f4/6a/f11147354d5bb4706fa5bb66aa4a95e5_.html]
nutch application_server 2015-05-27 14:22:17,823 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/f6/15/fa9abc11608e59b9191751e951199d36_.html]
nutch application_server 2015-05-27 14:22:17,951 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/a9/c8/ae8ab7e79f0fc158c686f2928340e33d_.html]
nutch application_server 2015-05-27 14:22:18,020 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/4d/32/4e18329fddfc853931340ecf224ae0d6_.html]
nutch application_server 2015-05-27 14:22:18,063 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/d9/f2/d23bd3bd9bd50b82f763d8172f23c7b4_.html]
nutch application_server 2015-05-27 14:22:18,110 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/c5/e0/c01d42d958433c9ce5ef70d600f8be92_.html]
nutch application_server 2015-05-27 14:22:18,172 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/ef/a5/e2bab135f4df2fe8a16749415fcc66a9_.html]
nutch application_server 2015-05-27 14:22:18,284 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/e3/cf/e59e9c743db7d86fca3d2bd9f45e961f_.html]
nutch application_server 2015-05-27 14:22:18,361 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/97/60/9400157a73f121aa69065dcf07566326_opinion.html]
nutch application_server 2015-05-27 14:22:18,383 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/1e/eb/1b85deb2eed10ad7ec1bb2efb1409ed8_.html]
nutch application_server 2015-05-27 14:22:18,419 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/c4/72/c7aad21f4049d3d77aa08c5f2dac1a99_letters-editor.html]
nutch application_server 2015-05-27 14:22:18,469 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/c6/45/c5d8da1d6b215b57420c076b5fa22bb1_.html]
nutch application_server 2015-05-27 14:22:18,531 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/f7/36/f58326fe7227a907302311906b4bade4_pets.html]
nutch application_server 2015-05-27 14:22:18,592 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/14/5a/1ef6be45401c0cd057517988ac58ec84_.html]
nutch application_server 2015-05-27 14:22:18,633 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/cd/19/ca5dc063decf967f10c52a46927e20b5_index.ssf]
nutch application_server 2015-05-27 14:22:18,675 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/8c/8c/80fc0b49c004c4f6814c0e84cc16c14c_.html]
nutch application_server 2015-05-27 14:22:18,731 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/a2/11/ac54f99625c5e71617e3dde617c4411e_recruiting.html]
nutch application_server 2015-05-27 14:22:18,785 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/30/85/394befaf0454be8f8c3341da5d32f4ef_.html]
nutch application_server 2015-05-27 14:22:18,840 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/1b/e5/175b7537b95ac3b7edf6732e505c695f_sec.html]
nutch application_server 2015-05-27 14:22:18,918 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/f0/f6/f27528e904770980f6fa93cc6132b039_sec-cocktail.html]
nutch application_server 2015-05-27 14:22:18,969 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/f2/7f/fb1fa2462066c5a873482d26f1b048be_.html]
nutch application_server 2015-05-27 14:22:19,002 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/d9/07/d103abad9012ed5d0969ab5475a13a0e_complete.html]
nutch application_server 2015-05-27 14:22:19,068 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/50/b2/5ff6b19b0c4ce79dbeb2aea024b20700_.html]
nutch application_server 2015-05-27 14:22:19,177 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/4f/76/44631507f376ca23799809b665eabfad_odds.html]
nutch application_server 2015-05-27 14:22:19,282 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/b1/bf/b22b22511af1330bbd45f8acf8014da3_affiliate_v002.css]
nutch application_server 2015-05-27 14:22:19,379 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/47/e4/40afe8fb7e58991be938e32f49001a37_adv_region.js]
nutch application_server 2015-05-27 14:22:19,412 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/f4/ea/f82747884d05731dee83596ba89e66ce_adv_toprail_v002.css]
nutch application_server 2015-05-27 14:22:19,441 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/1e/de/1d856a6aed7282e3d67462d6e2df1ce1_adv_common.js]
nutch application_server 2015-05-27 14:22:19,486 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/a2/ce/a4c2d0f22bab143bc9818834e276dfcb_adv_queryly.js]
nutch application_server 2015-05-27 14:22:19,529 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/76/f8/735abb2a60851766f21b432388c27fb6_adv_toprail_v002.js]
nutch application_server 2015-05-27 14:22:19,614 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/d3/fd/d6fa7d863c433b02f3b25747d43fe8d3_.html]
nutch application_server 2015-05-27 14:22:19,694 INFO tools.CommonCrawlDataDumper - Writing: [/vagrant/source/resources/crawls/commoncrawl-atf/19/cb/1eb23fe496f33f1fc60df91fbb0aab00_weddings.html]
nutch application_server 2015-05-27 14:22:19,711 INFO tools.CommonCrawlDataDumper - CommonsCrawlDataDumper File Stats:
nutch application_server TOTAL Stats:
nutch application_server [
nutch application_server {"mimeType":"image/vnd.microsoft.icon","count":"1"}
nutch application_server {"mimeType":"text/plain","count":"20"}
nutch application_server {"mimeType":"application/xhtml+xml","count":"11"}
nutch application_server {"mimeType":"application/octet-stream","count":"6"}
nutch application_server {"mimeType":"text/html","count":"54"}
nutch application_server ]
```

Add 2 buttons, one for each, to the UI.

**Comment by chrismattmann** [comment]
@ahmadia @brittainhard @lewismc @MJJoyce @nipunhere

**Comment by chrismattmann** [comment]
@shivikathapar

**Comment by chrismattmann** [comment]
Nipurn and Shivika, please take a look at this and how to implement it.

**Comment by chrismattmann** [comment]
![screen shot 2015-05-27 at 1 29 15 pm](https://cloud.githubusercontent.com/assets/395887/7842976/a4fa3f80-0474-11e5-8c17-4283f4e8bfbc.png)

Here's a screenshot - got the button in, now need to get the handler.

**Comment by ahmadia** [comment]
Fixed in #547
